fuzzyfinder.go

```diff
@@ -25,6 +25,7 @@ import (
 var (
 	// ErrAbort is returned from Find* functions if there are no selections.
 	ErrAbort   = errors.New("abort")
+	ErrCancel  = errors.New("cancel")
```

Comment:
I think that we don't need a self-defined cancellation error. Context is a standard mechanism, so how about returning` ctx.Err()` simply?